deployed.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I'd recommend the following fields:
```json
{
  "abi": "",
  "bytecode": "",
  "deployedBytecode": "",
  "address": "",
  "url": "",
}
```
Not sure what the "deployer" field should hold?